src/components/Application/ProgressBar.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Accordion, Container, Grid, Header, Label, List, Sticky } from 'semantic-ui-react'
+import { TemplateSectionPayload } from '../../utils/types'
+
+interface SectionPage {
+  sectionIndex: number
+  currentPage: number
+}
+
+interface ProgressBarProps {
+  serialNumber: string
+  templateSections: TemplateSectionPayload[]
+  sectionPage?: SectionPage
+}
+
+const ProgressBar: React.FC<ProgressBarProps> = ({
+  serialNumber,
+  templateSections,
+  sectionPage,
+}) => {
+  const childrenPanels = (section: string, totalPages: number) => {
```

Comment:
I think the naming of this one can improve, to describe that it's pages. i.e. pageList

Reply:
Agreed. Done!